app/models/highlight_fetcher.rb

```diff
@@ -38,5 +40,40 @@ def self.fetch_highlight(options = {})
         end
       end
     end
+
+    keep_session_in_db(client.session)
+    client
+  end
+
+  def self.keep_session_in_db(session)
+    puts "Storing cookie data into database"
+    ## data with Marshal.dump causes db error
+    data = session.driver.browser.manage.all_cookies
+    puts data.inspect
+    store = DataStore.where(key: key_for_cookie).first_or_create
+    store.update!(content: data)
+  end
+
+  def self.load_session_from_db(session)
+    if (store = DataStore.find_by_key(key_for_cookie)) && store.content.present?
+      puts "Loading cookie data from database"
+      ## visit is required before touching cookie
+      session.visit "https://www.#{AmazonInfo.domain}/" unless session.current_url =~ /\Ahttp/
```

Comment:
__[rubocop]__
* Performance/StartWith: Use `String#start_with?` instead of a regex match anchored to the beginning of the string. [:ref](https://github.com/JuanitoFatas/fast-ruby#stringmatch-vs-stringstart_withstringend_with-code-start-code-end)

<!-- Comment by [SideCI](https://sideci.com) -->